### DISCUSSION TITLE
Event markets available in website but unavailable in API

### DISCUSSION THREAD

**Discussion by jbn1981** [cloudbet/docs#80] [Q&A]
Hi,

In some events I can see markets available in the website but unavailable when using the API.

For example, the tennis event with id 23226858, I can see (and bet) "Winner" market using the website but when I obtain the event information using the API (service /v2/odds/events/23226858), the "tennis.winner" market is not available in the "markets" list obtained in the response, so I can't bet on it, ¿why is that?

I attach two files showing the problem.

Thanks.

![get-event-web](https://github.com/Cloudbet/docs/assets/32179115/554df142-4e7d-4b16-9852-ae371f209021)
![get-event-api](https://github.com/Cloudbet/docs/assets/32179115/4f00e2fe-b314-4175-a249-d69c0bb1a113)

**Comment by kgravenreuth** [comment]
Hi @jbn1981 ,

Thanks for reporting this issue. It should be fixed now and the missing tennis winner market should be available again via the Cloudbet API. 

As always, please refer to the full list of markets available via the Cloudbet API on this [Cloudbet markets github gist](https://gist.github.com/kgravenreuth/6703e1e213aecac4d5728f2f699d34e7#file-markets-json)

Thanks again for the details given in this issue, and let us know if there's anything else.

Regards,
Klaus Gravenreuth
Product & Engineering @ Cloudbet

**Comment by jbn1981** [comment]
Hi,

Thanks for the response.

I think I found another error in the API.

When querying odds for event 22842985 (/v2/odds/events/22842985), I can see in the response this:

{
     "outcome": "away",
     "params": "**handicap=-1.5**",
     "price": 1.813,
     "minStake": 0.1,
     "maxStake": 137.83112,
     "probability": 0.527,
     "status": "SELECTION_ENABLED",
     "side": "BACK"
},

But in fact, according to the website, "params" property should be "handicap=1.5", and not "handicap=-1.5".

Right?

I attach capture.

Thanks.

![cloudbet-api-error](https://github.com/Cloudbet/docs/assets/32179115/cb7a13df-c3e3-442d-8963-9a0a402e05fc)

**Reply by kgravenreuth** [reply]
Hi, this is explained in another one of our discussions on this topic: https://github.com/Cloudbet/docs/discussions/26

Basically, **The handicap value is dictated by the home team value and inverted on the away side.**

If you have any further questions on this, then please ask in #26 thanks!

Regards,
Klaus Gravenreuth
Product & Engineering @ Cloudbet
